cmd/earthly/build_cmd.go

```diff
@@ -188,7 +188,11 @@ func (app *earthlyApp) actionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs
 		if cloudClient.IsLoggedIn(cliCtx.Context) {
 			if app.logstreamUpload {
 				doLogstreamUpload = true
-				logstreamURL = fmt.Sprintf("https://ci.earthly.dev/todourl/%s", app.logbusSetup.InitialManifest.GetBuildId())
+				if strings.Contains(app.cloudGRPCAddr, "staging") {
```

Comment:
This is a hack... but I think it's good enough for now. Thoughts?

Reply:
This works for now.
The server may return this url as well at some point.